File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/controller/dto/OpprettBucOgSedDto.java
package no.nav.melosys.eessi.controller.dto;

import java.util.Collection;
import java.util.Collections;

import lombok.Data;
import no.nav.melosys.eessi.models.SedVedlegg;

@Data
public class OpprettBucOgSedDto {
    private SedDataDto sedDataDto;
    private Collection<SedVedlegg> vedlegg = Collections.emptySet();
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/controller/dto/OpprettBucOgSedDto.kt
package no.nav.melosys.eessi.controller.dto

import com.fasterxml.jackson.annotation.JsonSetter
import com.fasterxml.jackson.annotation.Nulls
import no.nav.melosys.eessi.models.SedVedlegg

data class OpprettBucOgSedDto(
    var sedDataDto: SedDataDto? = null,
    @JsonSetter(nulls = Nulls.AS_EMPTY)
    var vedlegg: Collection<SedVedlegg> = emptySet()
)
